DESIGN.md

```diff
@@ -117,23 +113,63 @@ ram_reserved = (256, 512, 1024) if total memory (4-7 GB, 8-16 GB, 16 GB+)
 
 > Any Replica should/would/will receive the same settings as the Primary Master, as a Replica is required to have the same system resources as the Primary Master.
 
-#### Large Reference Architecture (Compilers)
+#### Large Reference Architecture (Master and Compilers)
```

Comment:
This header is documenting, in parens, the role ... not describing the architecture.